tutorials/space_shooter/lib/steps/1_getting_started/1_flame_game/code.dart

```diff
@@ -4,7 +4,6 @@ import 'package:flutter/material.dart';
 
 class Player extends PositionComponent {
   static final _paint = Paint()..color = Colors.white;
-
```

Comment:
Unnecessary change and unrelated to this PR.

Reply:
This got changed while testing i fixed it now

Reply:
Ok sir